app/services/idv/flows/in_person_flow.rb

```diff
@@ -13,6 +38,10 @@ def initialize(controller, session, name)
       def self.session_idv(session)
         session[:idv] ||= { params: {}, step_attempts: { phone: 0 } }
       end
+
+      # def flow_session
+      #   {}
+      # end
```

Comment:
looks like this didn't end up getting used, let's remove
```suggestion
```

Reply:
fixed in 348ffb5b5c113ae2c76c9b05ded084599a7a7886